File: HackerRank Solutions/10 Days of Statistics/Day 5 Poisson Distribution II.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    double A, B;
    scanf("%lf%lf",&A,&B);
    double dailyCostA = 160 + (40 * (A + (A * A)));
    double dailyCostB = 128 + (40 * (B + (B * B)));
    printf("%.3lf", dailyCostA);
    printf("\n%.3lf", dailyCostB);
    return 0;
}

File: HackerRank Solutions/10 Days of Statistics/Day 5 Poisson Distribution II.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
A, B = [float(num) for num in input().split(" ")]
dailyCostA = 160 + (40 * (A + A**2))
dailyCostB = 128 + (40 * (B + B**2))
print(round(dailyCostA, 3))
print(round(dailyCostB, 3))
